content/docs/self-hosted-runners.md

```diff
@@ -339,7 +339,7 @@ credentials.
 </tab>
 </toggle>
 
-### Cloud Compute Resource Credentials
+## Cloud Compute Resource Credentials
```

Comment:
not sure about this change - better to keep under `Environment Variables` umbrella?

Reply:
with this tweak they show up in the sidebar and, I felt they were independent enough to warrant that:
![image](https://user-images.githubusercontent.com/1105843/171197845-749ede63-713b-4d7a-9f03-8b0e2b25e5dd.png)
But I don't have strong feelings so I can change it back if you still want it under the `Envionment Variables`